src/util/fs_helpers.cpp

```diff
@@ -41,6 +41,10 @@
 #include <shlobj.h> /* For SHGetSpecialFolderPathW */
 #endif // WIN32
 
+#ifdef __APPLE__
+#include <sys/mount.h>
```

Comment:
The manpage says we need `#include <sys/param.h>` too

https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man2/statfs.2.html

Reply:
added in df1ba101419729aafa382d970ee605e2a6273e26